### DISCUSSION TITLE
Using external Step implementation with Gauge/Java/Maven

### DISCUSSION THREAD

**Discussion by Barrick89** [getgauge/gauge#2616] [General]
Hi,

I hope someone can help me with my problem. We have a Java/Maven project where we included an external library as dependency that contains Gauge step implementations. When I try to use one of the steps in a spec file it first looks good as IntelliJ doesn't mark it red and I can jump directly from the step to the implementation in the external library. But when I try to execute the test case then I get:

Message: Step implementation not found

From this issue (https://github.com/getgauge/gauge-java/issues/236) I know that the problem existed a few years ago but that it was fixed but strangely I get the exact same error.

My versions:
Gauge version: 1.6.8
Commit Hash: afc97cb

Plugins
-------
html-report (4.3.1)
java (0.11.0)
screenshot (0.3.0)
spectacle (0.2.0)
xml-report (0.5.1)
xray-report (0.1.0)

IntelliJ Gauge Plugin: 231.8109.90
Gauge Maven: 1.6.3 
Maven 3.9.9

**Comment by chadlwilson** [comment]
It's a bit of a long-shot, but you might want to check that the gauge-java version defined in your Maven project is the same as the one for your plugin (i.e a new/recent version) - or at least not some old version.
